spec/broadside/ecs/ecs_deploy_spec.rb

```diff
@@ -176,25 +176,54 @@
     end
   end
 
-  describe '#run' do
-    let(:local_deploy_config) { { command: %w(run some command) } }
+  describe '#run_commands' do
```

Comment:
I realize this is a private method...but `run` does barely anything so I think `run_commands` should receive the specs